client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -17,7 +19,7 @@ import { getLatestAtomicTransfer } from 'calypso/state/atomic/transfers/selector
 import { pluginInstallationStateChange } from 'calypso/state/marketplace/purchase-flow/actions';
 import { MARKETPLACE_ASYNC_PROCESS_STATUS } from 'calypso/state/marketplace/types';
 import { fetchSitePlugins } from 'calypso/state/plugins/installed/actions';
-import { getPluginOnSite, isRequesting } from 'calypso/state/plugins/installed/selectors';
+import { getPluginOnSite } from 'calypso/state/plugins/installed/selectors';
```

Comment:
Just like above, we continue to retrieve plugins from Redux but we also use the `react-query` version here and there. What's the migration plan you have in mind?

Reply:
My plan was to ship this ( get the plugin data from the redux store for now) and then implement some sort of store for each of the using `react-query`.

Reply:
Gotcha. In that case, the problem is that we'll need to use the `use-plugins-query` query everywhere where we fetch plugins with Redux, otherwise there will be discrepancies. Is this something you were planning in this PR?